### PR TITLE
Translated to Croatian.

### PR DESCRIPTION
**Pull request by TheWildHorse**

### CODE REVIEW

File: lang.hr.php [hunk 1]
```diff
@@ -0,0 +1,60 @@
+<?php
+
+/*------------------
+Language: English
```
Comment (miyurusankalpa) [inline lang.hr.php]: can you change the language comment? thanks

**Comment by miyurusankalpa** [comment]
can you make that simple change? thanks btw 👍

**Comment by TheWildHorse** [comment]
Fixed, sorry about that, I haven't seen that comment when translating.

**Comment by miyurusankalpa** [comment]
this looks great. thanks 👍 #1 
have a great hacktoberfest 😃

**Pull request merged**